frontend/src/components/TransactionItem.tsx

```diff
@@ -52,6 +52,10 @@ function TransactionItem({ tx }: Props) {
   const [showDetails, setShowDetails] = React.useState(false);
   const type = tx.type;
 
+  const amountSatsWithFees = Math.floor(
+    type === "outgoing" ? (tx.amount + tx.feesPaid) / 1000 : tx.amount / 1000
```

Comment:
I already asked to remove the conditional, and you removed the Math.floor too. Please can you check again